.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0
+
+jobs:
+  format:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1
+    steps:
+      - checkout
+      - run:
+          name: "Install dotnet-format subcommand"
+          command: |
+            dotnet tool install -g dotnet-format
+      - run:
+          name: "Check the formatting of the project"
+          command: |
+            export PATH="$PATH:/root/.dotnet/tools"
+            dotnet format --check src/
+  build:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1
+    steps:
+      - checkout
+      - run:
+          name: "Build the project"
+          command: |
+            dotnet build --configuration release src/
+  test:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1
+    steps:
+      - checkout
+      - run:
+          name: "Run the tests"
+          command: |
+            dotnet test --configuration release src/Algolia.Search.Test
+  release:
+    docker:
+      - image: mcr.microsoft.com/dotnet/core/sdk:3.1
+    steps:
+      - checkout
+      - run:
+          name: "Build the NuPKG release artifact"
+          command: |
+            dotnet pack --configuration release src/Algolia.Search
+      - run:
+          name: "Release the new version"
```

Comment:
Like in Scala, it will run when you perform the following steps from the `master` branch:

```sh
git add CHANGELOG.md *.csproj
git commit -m 'chore: update version to X.Y.Z'
git tag X.Y.Z
git push
git push --tags
```

Reply:
That's awesome! Great job 👏